doc/api/http.md

```diff
@@ -564,6 +573,29 @@ Once a socket is assigned to this request and is connected
 
 Returns `request`.
 
+### request.socket
+<!-- YAML
+added: v0.3.0
+-->
+
+* `socket` {net.Socket}
+
+Reference to the underlying socket. `socket` is an object of type
```

Comment:
I don't think we need to duplicate the type in a separate sentence if it is already specified above.